gfop/foodcounts.py

```diff
@@ -0,0 +1,296 @@
+# foodcounts.py
+
+import numpy as np
+import pandas as pd
+from typing import List, Optional
+import os
+from importlib import resources
+import pkg_resources
+
+
+class FoodCounts:
+    def __init__(
+        self,
+        gnps_network: str,
+        sample_types: str,
+        all_groups: List[str],
```

Comment:
**todo:** These two variables can probably be renamed to be more descriptive. It was crappy naming from me when I wrote the code.